tests/test_no_todo.py

```diff
@@ -13,12 +13,51 @@ def test_no_todo(monkeypatch: pytest.MonkeyPatch) -> None:
         os.chdir(tmp_dir)
 
         with open("tmp.py", "w") as file:
-            print("print('hello world') # TODO: update", file=file)
+            print("print('hello world')", file=file)
 
-        with pytest.raises(SystemExit):  # Not updated the setup file so error
-            no_todos.main()
+        no_todos.main()  # No error
+
+
+def test_divide_by_todo(monkeypatch: pytest.MonkeyPatch) -> None:
+    """Checks that divisions are not confused with single-line comments"""
+    monkeypatch.setattr("sys.argv", ["no_todos.py", "tmp.py"])
+
+    with TemporaryDirectory() as tmp_dir:
+        os.chdir(tmp_dir)
 
         with open("tmp.py", "w") as file:
-            print("print('hello world')", file=file)
+            print(
+                """
+                todo = 1
+                done = 10 / todo
+                """,
+                file=file,
+            )
+
+        no_todos.main()  # No error
+
+
+def test_empty_comment(monkeypatch: pytest.MonkeyPatch) -> None:
+    """Checks that empty comments do not get flagged"""
+    monkeypatch.setattr("sys.argv", ["no_todos.py", "tmp.py"])
+
+    with TemporaryDirectory() as tmp_dir:
+        os.chdir(tmp_dir)
+
+        with open("tmp.py", "w") as file:
+            print("print('hello world') #", file=file)
```

Comment:
intriguing https://regex101.com/ in python mode is happy with this 🤷🏼

Reply:
ah, no, this test arose because of a mistake in my initial correction :)

Reply:
lovely. more tests = better